src/models/searchservice/response/DirectAnswerFieldType.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Type for value of DirectAnswer.
+ *
+ * @public
+ */
+export enum DirectAnswerFieldType {
```

Comment:
will this be used in `FieldValueDirectAnswer` for the generic type param?

Reply:
No, this will be used when we create specific direct answer types like when we create `UrlDirectAnswers`, `AddressDirectAnswers`.., we would use the corresponding field of `FieldValueDirectAnswer` as the `fieldType`